packages/plugins/Trader/src/SiteAdaptor/components/RouterDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import { InjectedDialog, type InjectedDialogProps } from '@masknet/shared'
+import { useLayoutEffect } from 'react'
+import { useLocation, useNavigate } from 'react-router-dom'
+import { RoutePaths } from '../constants.js'
+import { t } from '@lingui/macro'
+
+export function RouterDialog(props: InjectedDialogProps) {
+    const { pathname } = useLocation()
+    const navigate = useNavigate()
+
+    const titleMap: Record<RoutePaths, string | null> = {
+        [RoutePaths.Swap]: t`Exchange`,
+        [RoutePaths.History]: t`History`,
+        [RoutePaths.Confirm]: t`Confirm swap`,
+        [RoutePaths.SelectLiquidity]: t`Select Liquidity`,
+        [RoutePaths.Slippage]: t`Slippage`,
+        [RoutePaths.QuoteRoute]: t`Quote Route`,
+        [RoutePaths.TradingRoute]: t`Trading Route`,
+        [RoutePaths.Exit]: null,
+        [RoutePaths.NetworkFee]: t`Network fee`,
+        [RoutePaths.Transaction]: t`Transaction Details`,
```

Comment:
this is wrong. you should use `msg`, not `t`.

See https://lingui.dev/tutorials/react-patterns#picking-a-message-based-on-a-variable